test/tools/Modules/WebListener/WebListener.psd1

```diff
@@ -0,0 +1,10 @@
+@{
+    ModuleVersion = '1.0.0'
+    GUID = '90572e25-3f15-49b0-8f25-fb717d3ef46a'
+    Author = 'Mark Kraus'
+    CompanyName = ''
+    Copyright = ''
+    Description = 'An HTTP and HTTPS Listener for testing purposes'
+    RootModule = 'WebListener.psm1'
+    FunctionsToExport = @('Start-WebListener','Stop-WebListener', 'Get-WebListener', 'Get-WebListenerUrl','Get-TestClientCertificate')
```

Comment:
From my experience - It is better to put every name on separate line and sort - easy search, easy add new.